content/backend/typescript-apollo/1-getting-started.md

```diff
@@ -0,0 +1,464 @@
+import { ApolloServer } from "apollo-server";
+
+// 1
+import { schema } from "./schema";
+export const server = new ApolloServer({
+    schema,
+});
+
+const port = 3000;
+// 2
+server.listen({port}).then(({ url }) => {
+    console.log(`🚀  Server ready at ${url}`);
+});
+
+```
+
+</Instruction>
+
+Let's understand what's going on here by walking through the numbered comments: 
+
+1. The schema object you created using `nexus` defines your GraphQL schema. You need to provide this when instantiating your server.
+
+2. You start the server and specify the port. After the server starts, it returns a `url` string inside a promise. 
+
+Now that your server is ready let's give it a run and see what happens!
+
+<Instruction> 
+
+Open up your terminal and run the following command
+
+```bash(path=".../hackernews-typescript/")
+npm run dev
+```
+
+</Instruction>
+
+If everything goes well, you should see the following output inside the terminal:
+
+
+```(nocopy)
+🚀  Server ready at http://localhost:3000/
+```
+
+Now if you navigate to `http://localhost:3000/` you should see the following page: 
+
+![Apollo Server Landing Page](https://i.imgur.com/2bOq8c1.png)
+
+If you click on the `Query your Server` button, you will be redirected to The Apollo Studio Explorer. This is an _online web-based GraphQL IDE_ for running queries and exploring your schema (along with many other advanced features). 
+
+> **Note:** If you'd prefer an _offline_ IDE that does not need access to the internet, there's a section covering this at the bottom of this chapter. 
+
+![Apollo Studio Explorer](https://i.imgur.com/jsDqCAi.png)
+
+In the schema tab placed on the left (the first icon under the apollo logo), you can see the entire GraphQL schema. There's not much there right now, but in the future it will come in handy to explore the details of your schema. 
+
+![Schema Apollo Studio](https://i.imgur.com/fJK6FGq.gif)
+
+Now, you will run the `ok` Query.
+
+<Instruction> 
+
+Go back to the explorer tab (below the Schema tab) and type the following query in the `Operations` window. Then press the `Query` button.
+
+```graphql
+query Query {
+  ok
+}
+```
+
+</Instruction> 
+
+You should see a response like this: 
+
+```graphql(nocopy)
+{
+  "data": {
+    "ok": true
+  }
+}
+```
+
+![Run Ok Query](https://i.imgur.com/UVQVOTu.gif)
+
+Congratulations, you just implemented and successfully tested your first GraphQL query 🎉
+
+
+> **Note:** Quickly try something, make a quick change *anywhere* in your code. You will see the server restart again with a  message in your console log saying "Restarting: ...". This is because `ts-node-dev` will constantly check your code for any changes and update the server to reflect those changes. **From now on, you will always keep ts-node-dev running.** This will not only ensure that your server is always running, but it will also ensure nexus generates the most updated representation of your GraphQL SDL and types. 
```

Comment:
```suggestion
> **Note:** Quickly try something, make a quick change *anywhere* in your code. You will see the server restart again with a  message in your logging output saying `"Restarting: ..."`. This is because `ts-node-dev` will constantly check your code for any changes and update the server to reflect those changes. **From now on, you will always keep `ts-node-dev running`.** This will not only ensure that your server is always running, but it will also ensure Nexus generates the most updated representation of your GraphQL SDL and types. 
```

Reply:
I'd either not make this a not since it contains an instruction ("Quickly try something, make a quick change *anywhere* in your code") or make it clearer that this instruction is optional.